hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/INode.java

```diff
@@ -339,6 +339,13 @@ public boolean isFile() {
     return false;
   }
 
+  public boolean isSetStoragePolicy() {
```

Comment:
A minor comment. It will be good if we add a comment to this method. People may confuse about "inheriting storage policy from its parent" and "containing storage policy itself". Add a comment like getStoragePolicyID() and getLocalStoragePolicyID().

Reply:
@wojiaodoubao thank you very much for your comment. I  had fixed  it and repush it.